frontend/src/css/collapsing-item.css

```diff
@@ -0,0 +1,24 @@
+/*
+collapsing-item.css is used for the CollapsingItemContainer.jsx styles that cannot be added inside a const
+*/
+
+.collapsing-item-container:hover .expand-icon {
+  color: white !important;
+}
+
+.expanded-button-style {
+  border-radius: 5px 5px 0 0;
+}
+
+.expand-icon-style {
+  color: white !important;
```

Comment:
Same thing with this icon.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/171/commits/2f4ef15f0b6d42db14e88a8c65e608f84e625f42 (see in PR https://github.com/code-for-canada/project-thundercat/pull/171)